ecdsa_test.go

```diff
@@ -70,7 +70,11 @@ func testECDSASignAndVerify(t *testing.T, c elliptic.Curve) {
 	if !openssl.VerifyECDSA(pub, hashed[:], signed) {
 		t.Errorf("Verify failed")
 	}
-	signed[0] ^= 0xff
+	// Alter the last byte of the signature to make it invalid
+	// without corrupting the DER encoding, which would cause
+	// some OpenSSL providers, such as SymCrypt-OpenSSL, to write
+	// a noisy warning to stderr.
```

Comment:
IMO a comment for a change like this often looks good in PR, but can be unclear for future readers because they expect that they're reading an explanation for the entire next line. Suggestion:

```suggestion
	// Alter the signature to intentionally make it invalid. Change the last
	// byte (rather than the first) to avoid corrupting the DER encoding, which
	// would cause some OpenSSL providers, such as SymCrypt-OpenSSL, to write a
	// noisy warning to stderr.
```